packages/instant/package.json

```diff
@@ -7,7 +7,6 @@
     "private": true,
     "description": "0x Instant React Component",
     "main": "umd/instant.js",
-    "private": true,
```

Comment:
we want to keep this private

Reply:
`private` was defined twice, see line 7 above ^